app/build.gradle

```diff
@@ -89,6 +90,8 @@ dependencies {
       'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.2.1',
       'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.2.1',
       'org.mockito:mockito-core:2.7.22',
+      'com.google.firebase:firebase-analytics-ktx:17.4.2',
```

Comment:
Nit: We are actually following alphabetical order here.

Reply:
Done